README.md

```diff
@@ -2,12 +2,37 @@
 
 This is a small simple service to mirror hex.pm to the local disk.
 
-It was written with [elixir.camp](http://elxiir.camp), where there is no internet connection, and so needs to be able to work completely offline.
+This was written for [elixir.camp](http://elxiir.camp),
+where there is no internet connection, and so needs to be
+able to work completely offline.
 
 ## Instructions
 
 1. Download it, setup deps and run
-2. Wait. By default it will start mirroring automatically and save to `./tarballs` As of July 2016, this will fetch ~700MB of data
+
+```bash
+git clone git@github.com:aussiegeek/hex_mirror.git
+npm install
+mix deps.get
+npm install
+mix phoenix.server
+```
+
+2. Wait. By default it will start mirroring automatically and save to
+`./tarballs`. As of July 2016, this will fetch ~700MB of data.
+
+3. People that want to use your mirror will need to do this
+(assuming your local address is 'thingy.local'):
+
+```bash
+mix hex.config mirror_url http://thing.local
```

Comment:
Yeah, I noticed that, too. Still didn’t work though.


> On 24 Aug 2016, at 4:26 PM, Alan Harper <notifications@github.com> wrote:
> 
> In README.md:
> 
> > +```bash
> > +git clone git@github.com:aussiegeek/hex_mirror.git
> > +npm install
> > +mix deps.get
> > +npm install
> > +mix phoenix.server
> > +```
> > +
> > +2. Wait. By default it will start mirroring automatically and save to
> > +`./tarballs`. As of July 2016, this will fetch ~700MB of data.
> > +
> > +3. People that want to use your mirror will need to do this
> > +(assuming your local address is 'thingy.local'):
> > +
> > +```bash
> > +mix hex.config mirror_url http://thing.local
> 
> I made the same mistake. I'm not sure if it's a bug, but HEX_UNSAFE_REGISTRY needs to be set to 0
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>